contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -78,14 +79,18 @@ contract StakeManager is
         address _validatorShareFactory,
         address _governance,
         address _owner,
-        address _extensionCode
+        address _extensionCode,
+        address _tokenLegacy,
+        address _migration
```

Comment:
why are we appending the new state in the existing initializer? we've got `initializeLegacy` below to set migration and new token address. also, this will revert with 'already init' afaik

Reply:
I think the reinitialize function was added in a recent upgrade and the variables introducded there were also added to the initialize function. So I did the same.
I guess it's useful should anyone ever want to do a new deployment.
If you think it's uneeded extra I'll remove it.

Reply:
ah gotcha, so we don't plan to call this method in this upgrade! i would still recommend we remove this here, especially for this upgrade bc in case anyone freshly forks these contracts for their consensus they would ideally want to remove tokenLegacy instead of inheriting that tech debt

Reply:
I would agree. I will remove them.